.github/workflows/release.yml

```
@@ -218,6 +218,17 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build and push docker image
+        if: ${{ contains(needs.prepare.outputs.tag_name, 'preview') }}
+        uses: docker/build-push-action@v3
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)